easybuild/scripts/bootstrap_eb.py

```diff
@@ -448,7 +459,9 @@ def main():
     for path in orig_sys_path:
         include_path = True
         # exclude path if it's potentially an EasyBuild package
-        if 'easybuild' in path:
+        if 'easybuild' in path or os.path.exists(os.path.join(path, 'easybuild')):
```

Comment:
excluding any entries in `sys.path` that look like they have something to do with EasyBuild

it's OK if we exclude more (since we don't rely on stuff being there already), but we can't easily drop the entire `sys.path`